index.html

```diff
@@ -21,27 +21,218 @@
+                        <li><a href="#" ><img src="images/angellist.svg"/></a></li>
+                    </ul>
+                </div>
             </div>
-            <div id="main-item2">
-                <p>
-                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hesitate to contact me.
-                </p>
-            </div>
-            <div id="main-item3">
-                <ul><li><a href="https://twitter.com/i/flow/login" ><img src="images/twitter.svg"/></a></li>
-                    <li><a href="https://www.linkedin.com/in/kehinde-aminah-h/" ><img src="images/Linkedin icon.svg"/></a></li>
-                    <li><a href="https://medium.com/@Armynerh" ><img src="images/medium.svg"/></a></li>
-                    <li><a href="https://github.com/Armynerh" ><img src="images/github.svg"/></a></li>
-                    <li><a href="#" ><img src="images/angellist.svg"/></a></li>
-                </ul>
+        </section>
+        <section>
+            <div id="about-container">
+            <div id="works-div">
+                <h2>My Recent Works</h2>
             </div>
         </div>
-    </main>   
+            <div class="about-header">
 
+            </div>
+            <div class="grid-container">
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="grid-items">
+                    <div class="grid-card">
+                        <div class="post-card">
+                            <h4>Multi-Post Stories Gain+Glory</h4>
+                        </div>
+                        <div class="post-list">
+                            <ul>
+                                <li>Ruby on Rails</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                                <li>HTML</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- Kindly make sure your design stick as much as possible to the  Figma design as  required in the project, so I suggest you reduce the size of the text and adjust it a bit the disposition to make it stick to the Figma design and do it for all the cards :+1: 
- as it is shown in the Figma design the background images are not also well arranged. Kindly  make  sure the background of the home page does not appear on all pages it will make your design stick more to the Figma design  :+1: 

## Your design 

![image](https://github.com/Armynerh/personal_portfolio/assets/95297251/6c4bac83-21a7-4b7d-b773-4eb2f680e5da)

## Figma design 
![image](https://github.com/Armynerh/personal_portfolio/assets/95297251/f5bc8d2f-7192-456d-8e3b-ef20b755893c)